migrate.js

```diff
@@ -77,7 +76,7 @@ const settingsFileName = (mode, forum) => {
     mode = "dev";
   } else if (mode === "production") {
     mode = "prod";
-  } else if (!isRunCommand) {
+  } else if (!["up", "down", "pending", "executed"].includes(command)) {
```

Comment:
Just checking that everywhere else `isRunCommand` gets used is explicitly not being updated

Reply:
Yup that is on purpose - the two new commands here seemed to work without them so I just changed this line